build.gradle

```diff
@@ -60,7 +60,9 @@ jib {
 dependencies {
     implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
 
-    testImplementation group: 'junit', name: 'junit'
+    testImplementation('org.springframework.boot:spring-boot-starter-test') {
+        exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
```

Comment:
Not need to have a testing framework. Let's go with plane Mockito or what ever comes with Spring.

Reply:
that's a good point , aren't you going to do integration test for spring boot ?

Reply:
we will and at that time we might need to add some dependencies and we are not sure which library we should pick at that time. for now lets not worry about that.